vars/piperStageWrapper.groovy

```diff
@@ -80,7 +80,7 @@ private void executeStage(script, originalStage, stageName, config, utils, telem
         def body = originalStage
 
         // First, check if a global extension exists via a dedicated repository
-        if (globalExtensions) {
+        if (!Boolean.valueOf(env.PIPER_NO_EXTENSIONS) && globalExtensions) {
```

Comment:
Personally, for me `! ...(env.PIPER_DISABLE_EXTENSION..` would be easier to understand than `! ...(env.PIPER_NO_EXTENSION..`.

Reply:
I agree.